### PR TITLE
Add `rake db` task to automatically set up local database based on Heroku

### PR DESCRIPTION
**Pull request by KrauseFx**
This will trigger a backup, download it, create a local database if necessary and import all the data

### CODE REVIEW

**Comment by ohayon** [comment]
👍

File: Rakefile [hunk 1]
```diff
@@ -4,3 +4,18 @@
...
 
 Rails.application.load_tasks
+
+# Create a new DB backup, and import it to the local Postgres DB
+task :db do
+  app = ENV["HEROKU_APP"] || "fastlane-refresher"
+  db_name = ENV["DB_NAME"] || "refresher"
+
+  puts "This script is going to drop your local database #{db_name} and fetch the database from heroku #{app}. Please confirm!"
+  STDIN.gets
```
Comment (mfurtak) [inline Rakefile:14]: I think someone might reasonably expect that if they entered 'n' here, that things would stop.

Should the warning be changed to say "kill the process now" or similar? Or should 'y', 'n' be the only valid inputs?

Comment (KrauseFx) [reply Rakefile:14]: Yes, that would be nicer, however I didn't initially want to spend the time looking for how to do that, as we don't have highline or similar as dependency, related https://twitter.com/KrauseFx/status/775499416878788609 👍

Comment (KrauseFx) [reply Rakefile:14]: The user still has enough time to ctlr + c since creating the db, and downloading it takes a while

Comment (mfurtak) [reply Rakefile:14]: I agree that there's time, but I think the message printed out might convince someone to try to answer "n", but wind up dropping the database anyway.

If we don't want to add extra handling, I'd just change the message to end with something like "Quit now if that doesn't sound good, or press any key to continue."

**Pull request merged**